servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -69,183 +69,175 @@ void RenderForwardMobile::_map_forward_id(ForwardIDType p_type, ForwardID p_id,
 
 /* Render buffer */
 
-void RenderForwardMobile::RenderBufferDataForwardMobile::clear() {
-	if (color_msaa.is_valid()) {
-		RD::get_singleton()->free(color_msaa);
-		color_msaa = RID();
+void RenderForwardMobile::RenderBufferDataForwardMobile::free_data() {
```

Comment:
This appears to be leading to the crash caught by CI. Not sure what is going on, but it should be fixed before merging

Reply:
I'll have a look, I did some last minute changes here so I must have missed it being called before configure was called the first time (or indeed, being called from configure for the first time).

Reply:
Ah forgot to default a pointer to `nullptr`, so lovely that MSVC does this automatically so it's not habitual yet...